guide/C/ch_currency.xml

```diff
@@ -105,19 +108,19 @@ TODO -->
       <para>Since in this example you mostly work in USD, all of the parent accounts are set to USD. Of course,
         if you mostly work in Euros, you could change the currency of these parent accounts to EUR.
         The totals shown in the account tree window will always be converted to the currency of each
-        particular account. Notice, we also set up 3 Starting Balances equity accounts, used to
+        particular account. Notice, we also set up 3 Opening Balances equity accounts, used to
         initially populate the 3 banks.
       </para>
 
       <note>
-        <para>You could also set up just a single Starting Balance account and use a currency transfer to populate
+        <para>You could also set up just a single Opening Balance account and use a currency transfer to populate
```

Comment:
It's the case generally.  You need the basis cost of any non-home-currency assets in order to be able to have a balanced book with correctly recorded gains when you dispose of those assets. The correct way is to compute the basis of the asset and create a home-currency opening balance for it that you immediately transfer at the basis price to the actual asset account. For fixed assets that should be followed with an accrued depreciation transaction so that the book reflects the current tax value of the asset.